runtime/src/bank.rs

```diff
@@ -752,6 +758,29 @@ impl Bank {
 
     /// Create a new bank that points to an immutable checkpoint of another bank.
     pub fn new_from_parent(parent: &Arc<Bank>, collector_id: &Pubkey, slot: Slot) -> Self {
+        Self::_new_from_parent(
+            parent,
+            collector_id,
+            slot,
+            &mut None::<fn(&RewardCalcEvent)>,
+        )
+    }
+
+    pub fn new_from_parent_with_tracer(
```

Comment:
Same comment as above, I'd prefer a more descriptive name for `tracer`: `reward_calc_tracer`?

Reply:
41c8d0bb11